src/install/crc-install.ts

```diff
@@ -87,6 +145,50 @@ export class CrcInstall {
     }
   }
 
+  async hasUpdate(version: CrcVersion): Promise<CrcUpdateInfo> {
+    const latestRelease = await this.downloadLatestReleaseInfo();
+    this.crcCliInfo = new CrcCliInfoStorage(this.storagePath);
+    await this.crcCliInfo.loadInfo();
+    if (
+      compare(latestRelease.version.crcVersion, version.version, '>') &&
+      this.crcCliInfo.ignoreVersionUpdate !== latestRelease.version.crcVersion
+    ) {
+      return { hasUpdate: true, newVersion: latestRelease, currentVersion: version.version };
+    }
+
+    return { hasUpdate: false, currentVersion: version.version };
+  }
+
+  getUpdatePreflightChecks(): extensionApi.InstallCheck[] | undefined {
+    const installer = this.getInstaller();
+    if (installer) {
+      return installer.getUpdatePreflightChecks();
+    }
+    return undefined;
+  }
+
+  async doUpdate(
+    provider: extensionApi.Provider,
+    updateInfo: CrcUpdateInfo,
+    logger: extensionApi.Logger,
+  ): Promise<void> {
+    const answer = await extensionApi.window.showInformationMessage(
+      `You have ${productName} ${updateInfo.currentVersion}.\nDo you want to update to ${updateInfo.newVersion.version.crcVersion}?`,
+      'Yes',
+      'No',
+      'Ignore',
+    );
+    if (answer === 'Yes') {
+      await this.getInstaller().update(updateInfo.newVersion, logger);
+      const crcVersion = await getCrcVersion();
+      provider.updateDetectionChecks(getCrcDetectionChecks(crcVersion));
+      provider.updateVersion(crcVersion.version);
+      this.crcCliInfo.ignoreVersionUpdate = undefined;
+    } else if (answer === 'Ignore') {
+      this.crcCliInfo.ignoreVersionUpdate = updateInfo.newVersion.version.crcVersion;
```

Comment:
Is there a way to still install this update?
People might ignore it, meaning `ignore for now`.

Reply:
They have `Cancel` button.
I could rename `Ignore` to `Ignore %{version}`.

Reply:
This does make sense if expressed this way.

Reply:
let's put this in a new issue... but not implement this yet.
as I see this is also in the `answer ===` check